components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/gen/java/org/wso2/carbon/identity/api/server/idp/v1/IdentityProvidersApiService.java

```diff
@@ -60,6 +60,8 @@ public interface IdentityProvidersApiService {
 
       public Response deleteIDPTemplate(String templateId);
 
+      public Response exportIDP(String identityProviderId, Boolean exportSecrets, String accept);
```

Comment:
any reason why we can't use primitive type boolean here?

Reply:
This is auto generated from the swagger definition where the type of `exportSecretsQueryParam` is set as `type: boolean`.